docs/contributing/minutes.md

```diff
@@ -4,6 +4,103 @@ title: Team Minutes
 sidebar_label: Team Minutes
 ---
 
+## 30-03-2022
+
+### Present
+
+Sally, Greg, Daniel, Mitch
+
+Agenda Items
+
+### Sanity check Fundamentals
+
+What is the project?
+Week 1: ship personal home page (from intro to digital)
+Week 1: imaginary user stories for imaginary project
+Week 2: defining user stories for own website
+Week 2: choose a user story 
+Week 3: do some more of it
+
+We floated the idea of a "shop" page
+
+Step 1) make what we have make sense, let’s start with 
+
+@Greg: Make a ticket for making a project
+@Daniel: we can make a shop -> make a ticket for it
+@Sally: consolidate current version to make sense, based on improving the personal home page
```

Comment:
```suggestion
- @Sally: consolidate current version to make sense, based on improving the personal home page
```

Reply:
erm, lol. Actually maybe it's quicker just to say: your line breaks aren't showing up in markdown so would it be ok if you either:

- write
- lists

Or use full stops and capital letters? It's a bit hard to read.

Reply:
I took the liberty of list-ifying some bits to make it easier to read when I was reviewing. Hope that's ok with you @gregdyke!